### ISSUE TITLE
A few questions

### ISSUE THREAD

**Issue by rjbprime** [noahlange/homebrewery-electron#9]
Hi @noahlange,
I have a couple of queries re this fork.
1. Is it going to be updated to latest homebrewery code?
2. I've been trying to get this to run on Windows 10, and it loads, but unable to work out how to get the brew to save. Do I need to follow the instructions of the main homebrewery local install, then run this? Or is this separate from a local homebrewery install?

**Comment by noahlange** [comment]
Hey! Thanks for taking a look.

I haven't really touched this since the initial commits. I've been juggling a dozen other projects. Let me take a look and I'll let you know. 🙂

**Comment by rjbprime** [comment]
Cheers for that. Been looking for a way to use an offline homebrewery, as I'd like to be able to use it without net.

On a related note, what format does this save brew in on the local system? When I looked at it, the drop down in the File > Save menu said "All Files \*.\*". and nothing else, so I am unsure what to type for the file extension.

**Comment by noahlange** [comment]
It's plaintext, so it doesn't really matter; I'd recommend Markdown.

**Comment by rjbprime** [comment]
Any updates on this?

**Comment by noahlange** [comment]
https://brewie.noahlange.com

**Comment by noahlange** [comment]
Yes, actually! I've made some pretty substantial progress on the layout library and refactored most of the application into a standalone web-app — it'd be fairly simple to get this back into Electron.

The rebuild is much less opinionated in its styling; the hope is to create individual themes that cover the same ground as the Homebrewery. Initial preview is here: https://brewie.noahlange.com

**Comment by rjbprime** [comment]
I know it's been nearly a year but is there any updates on this?

**Comment by noahlange** [comment]
Howdy! Latest incarnation is [here](https://github.com/noahlange/markwright-app).

It doesn't include built-in Homebrewery themes, and it's rather WIP, but it might be a starting point.
